updater/src/Updater/Command.purs

```diff
@@ -80,12 +84,22 @@ runGenerate opts = do
       , displayName: fromMaybe ("`" <> spago.name <> "`") opts.displayName
       , displayTitle: fromMaybe (toTitleCase spago.name) opts.displayTitle
       , maintainers: map maintainerTemplate opts.maintainers
+      , usesJS: opts.usesJS
       }
 
-  runBaseTemplates variables
+    validatedTemplates =
+      maybe
+        (Right allTemplates)
+        (validateFiles { usesJS: opts.usesJS, templates: allTemplates })
+        opts.files
 
-  when opts.usesJS do
-    runJsTemplates variables
+  case validatedTemplates of
+    Right templates -> do
+      runTemplates variables templates
```

Comment:
I know that it's a bit clunky to move it here, but this would be the place to only run `appendReleaseInfoToChangelog` if the changelog was in the templates included by the user.

Reply:
Well, it kinda makes sense that it should be done here. I can make the chanhes